src/project/local/NodeFsLocalFile.ts

```diff
@@ -47,6 +48,10 @@ export class NodeFsLocalFile extends AbstractFile implements LocalFile {
         }
         return Promise.resolve(this);
     }
+
+    public isExecutable(): Promise<boolean> {
+        return fs.access(this.realPath, fs.constants.X_OK).then(_ => true).catch(_ => false);
```

Comment:
why those underscores? if you don't need use the variables use `()` instead. `_` for throw-away parameters is not idiomatic for JS.